src/dbnode/storage/bootstrap.go

```diff
@@ -138,6 +155,8 @@ func (m *bootstrapManager) Bootstrap() (BootstrapResult, error) {
 	m.mediator.DisableFileOpsAndWait()
 	defer m.mediator.EnableFileOps()
 
+	asyncResult.bootstrapStarted.Done()
+
 	// Keep performing bootstraps until none pending and no error returned.
 	var result BootstrapResult
```

Comment:
Perhaps we can use a defer here? Just in case another code path returns early.
```suggestion
	// Keep performing bootstraps until none pending and no error returned.
	var result BootstrapResult

	asyncResult.bootstrapStarted.Done()
	defer func() {
		asyncResult.bootstrapResult = result
		asyncResult.bootstrapCompleted.Done()
	}()
```

Reply:
(Would need to remove the bootstrapResult = result assignment and bootstrapCompleted.Done() calls at bottom of method obviously)